client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -31,6 +33,48 @@ const ECommerceTrialPlansPage = ( props: ECommerceTrialPlansPageProps ) => {
 		return getWooExpressMediumFeatureSets( { translate, interval } );
 	}, [ translate, interval ] );
 
+	const performanceOnlyOption = (
+		<ECommercePlanFeatures
+			interval={ interval }
+			monthlyControlProps={ { path: plansLink( '/plans', siteSlug, 'monthly', true ) } }
+			planFeatureSets={ wooExpressMediumPlanFeatureSets }
+			siteSlug={ siteSlug }
+			triggerTracksEvent={ triggerTracksEvent }
+			yearlyControlProps={ { path: plansLink( '/plans', siteSlug, 'yearly', true ) } }
+		/>
+	);
+
+	const mediumPlan = getPlans()[ PLAN_WOOEXPRESS_MEDIUM ];
+	// const smallPlan = getPlans()[ PLAN_WOOEXPRESS_SMALL ];
+	const mediumFeaturesRaw = [
+		...( mediumPlan?.getInferiorFeatures?.() || [] ),
```

Comment:
Maybe unrelated to this PR, but `getInferiorFeatures` as a name is confusing! I thought it was a typo for `getInteriorFeatures` until I looked it up. Why would our plans offer inferior features? 😆

Reply:
Yeah, that's a strange name. [Looks like](fbhepr%2Skers%2Spnylcfb%2Scnpxntrf%2Spnylcfb%2Qcebqhpgf%2Sfep%2Scynaf%2Qyvfg.gfk%3Se%3Q3sosp976-og) that's been the convention for `plans-list.tsx` for a long time though. 🤷

Reply:
Just a comment that may make sense (?)

```
	/**
	 * Features that are superseded by another feature included in this plan.
	 *
	 * Example: if a plan has 1TB of storage space,
	 * a feature for 20GB of storage space would be inferior to it.
	 */
         getInferiorFeatures?: () => Feature[];
```